playground/src/routes/index.svelte

```diff
@@ -23,13 +23,7 @@
             name
             entity
             enjoys
-            image {
-              srcset
-              srcsetwebp
-              srcsetavif
-              placeholder
-              aspectratio
-            }
```

Comment:
These fields won't show as part of the GraphQL schema but they'd be invisibly added. I think we should avoid these types of field indirections and add all queryable fields we surface as part of the schema.

Was this leading to a bug? I'd be happy to lend a spelunking hand - I believe we should be able to use `graphql-compose` to create this type object & subfields

Reply:
normally I would agree, but in the case of these the fields will change based on the config that is provided to svimg and they are always expected to be dumped into the webcomponent s-img...

I looked further into fragments and realized that fragments are client side only so that doesn't really work here,  I'm still not sure if JSON is a good idea or not though

Reply:
@tonyketcham given the context, what do you think? JSON or explicit fields?

Reply:
We might be able to dynamically generate the fields based on the passed in config. A parent resolver on `image` could run `svimg`, and then pass the results to the child field resolvers to access and return. The data could then be asked for on-demand, piecemeal if someone didn't want to use the s-img webcomponent.

A method we could try is merging in custom resolvers via https://graphql-compose.github.io/docs/api/SchemaComposer.html#merge

Reply:
Ah, I didn't catch this last comment before, it was a good idea